exercises/practice/darts/bin/fetch-cobolcheck

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# This file is a copy of the
```

Comment:
The fetch scripts might also need to be updated to match the other exercises.

Reply:
Oops